### PR TITLE
Migrate unique Heatmap controls

### PR DESCRIPTION
**Pull request by villebro**
### CATEGORY

Choose one

- [ ] Bug Fix
- [ ] Enhancement (new features, refinement)
- [x] Refactor
- [ ] Add tests
- [ ] Build / Development Environment
- [ ] Documentation

### SUMMARY
Move Heatmap specific controls from `controls.js` to `Heatmap.js`.

### ADDITIONAL INFORMATION

- [ ] Has associated issue: #9187
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### REVIEWERS
@rusackas @kristw

### CODE REVIEW

**Comment by villebro** [comment]
@kristw @rusackas There seems to be a bunch of controls that are used in two plugins but aren't really general purpose plugins, e.g. `normalized`, `show_values`, `global_opacity` etc. I propose we move these into the most appropriate "parent" plugin, an import it from there where needed. For example, `normalized` should probably be defined in `Histogram`, and can then be imported and customized in `Heatmap`.

File: superset-frontend/src/explore/controlPanels/Heatmap.js [hunk 1]
```diff
@@ -18,6 +18,14 @@
...
 import { t } from '@superset-ui/translation';
 import { nonEmpty } from '../validators';
+import { formatSelectOptionsForRange } from '../../modules/utils';
+
+const sortAxisChoices = [
+  ['alpha_asc', t('Axis ascending')],
+  ['alpha_desc', t('Axis descending')],
+  ['value_asc', t('Metric ascending')],
+  ['value_desc', t('Metric descending')],
+];
```
Comment (villebro) [inline superset-frontend/src/explore/controlPanels/Heatmap.js]: Note the added i18n and `sum(value)` -> `Metric`.

**Comment by kristw** [comment]
@villebro I am proposing each chart maintain their own copy of the control config to keep them independent. There are no value added for histogram and heatmap sharing the same `normalized` field when it is not clear if it indicates the same behavior on both charts.

**Comment by rusackas** [comment]
> @villebro I am proposing each chart maintain their own copy of the control config to keep them independent. There are no value added for histogram and heatmap sharing the same `normalized` field when it is not clear if it indicates the same behavior on both charts.

Following up here to mirror a Slack discussion... we can keep this pass to pulling out the singletons, but in another pass, we can start copying multiple-use controls into individual plugins, and removing the overrides that plugins are layering on. I propose continuing our per-plugin singleton-removal pass, and then doing a per-control pass next to start chewing away at the remaining list.

**Comment by villebro** [comment]
Ok, I'm cool with that, singleton pass it is 👍

**Review by rusackas — APPROVED** [review]
LGTM

**Pull request merged**